Profiles/Default Clean/Default Clean.json

```
@@ -0,0 +1,394 @@
+{
+  "Version": "v0.0.1",
+  "Name": "Default",

```

This needs to be "Default Clean" to match the name of the json file
```suggestion
  "Name": "Default Clean",
```